canonical-web-and-design/ubuntu.com
# Updates infra pricing table and amount of free storage to 192TB

## Done

- Updates infra pricing table with new collumn
- Updates free storage quantity from 92TB to 192TB on English and Japanese service description

## QA

- Go to /infra/pricing and check the tabe matches the table in the [copydoc](https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit#)
- Go to /legal/ubuntu-advantage-service-description (jp and eng) and check the amount of free storage is 192TB

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5457
